### DISCUSSION TITLE
use_gpu=True不报错，也没有反应，但是使用cpu就可以识别

### DISCUSSION THREAD

**Discussion by wangxun6** [PaddlePaddle/PaddleOCR#12839] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win10
- 版本号/Version：Paddle：paddlepaddle-gpu 2.5.1  PaddleOCR：paddleocr2.6.1.3
- 运行指令/Command Code：检测扫描形成的pdf另存为txt文本文档
- 完整报错/Complete Error Message：

目前就是我在调paddleocr时， ocr = PaddleOCR(use_angle_cls=True, lang="ch", use_gpu=False, show_log=False)，程序可以识别出来文本内容，并转存为txt文件。
但是 ocr = PaddleOCR(use_angle_cls=True, lang="ch", use_gpu=True, show_log=False)时，程序就无输出。
我的显卡是NVIDIA RTX A5000

**Comment by HengeLiu** [comment]
我也遇到了同样的问题，CPU可以正常输出，切换use_gpu=true后，没有任何识别结果，只是创建了结果文件夹。
我看了代码发现是版面识别后，所有box的confidence都是0或接近于0，导致版面识别的结果输出为空，从而没有任何后续识别过程导致。
请问您解决了吗？

**Comment by liwenli233** [comment]
我也是同样的问题，gtx750

**Comment by SWHL** [comment]
可以尝试更换新一些的显卡试试

**Comment by cyy0523xc** [comment]
同遇到问题，启动的时候，就像是卡住了，也没有错误日志。测试在A5000和A6000都有问题，在V100、T4、2080TI上则能正常

**Reply by GreatV** [reply]
更新一下paddle版本

**Reply by cyy0523xc** [reply]
![ocr](https://github.com/PaddlePaddle/PaddleOCR/assets/1209330/15aad972-d86d-481f-bdf9-4fb8cb5d241c)
@GreatV 从2.0.6升级到了2.5.0.3，启动的时候还是卡在这里了

**Reply by GreatV** [reply]
可以安装一下 dev 版本的paddle

**Reply by cyy0523xc** [reply]
刚升级的是paddleocr版本，paddle版本还没试过

**Reply by cyy0523xc** [reply]
> 可以安装一下 dev 版本的paddle

把paddle从2.1.3升级到2.4.2，把paddleocr升级到2.7.3，启动的时候还是一样卡在那里

**Reply by GreatV** [reply]
paddle现在的版本是 3.0 beta，你可以试试
